.github/workflows/check.yml

```diff
@@ -0,0 +1,60 @@
+---
+
+name: Check Set-Up & Build
+
+# Controls when the action will run.
+# yamllint disable-line rule:truthy
+on:
+  # Triggers the workflow on push or pull request events
+  # but only for the manta (default) branch.
+  pull_request:
+    branches: [manta-pc]
+  push:
+    branches: [manta-pc]
+
```

Comment:
Hmm, we shouldn't disable it for the ```manta``` channel.

Reply:
Now, we are having two branches: `manta` and `manta-pc`, both will not be merged in the near future.